content/docs/concepts/remote_write_spec_2_0.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Prometheus Remote-Write Specification 2.0
+sort_rank: 4
+---
+
+# Prometheus Remote-Write Specification
+
+* Version: 2.0
+* Status: Proposed
+* Date: May 2024
+
+The remote write specification, in general, is intended to document the standard for how Prometheus and Prometheus remote-write-compatible agents send data to a Prometheus or Prometheus remote-write compatible receivers.
+
+This document is intended to define a second version of the [Prometheus Remote Write](./remote_write_spec.md) API with minor changes to protocol and semantics. This second version also adds a new wire format with new features enabling more use cases and wider adoption on top of performance and cost savings. Finally, this spec outlines how to implement backward compatible senders and receivers (even under a single endpoint) using existing basic content negotiation request headers. More advanced, automatic content negotiation mechanisms might come in future versions, if needed. For the rationales behind the 2.0 specification, see [the formal proposal](https://github.com/prometheus/proposals/pull/35).
```

Comment:
Good point. I think yes, because in this version we also publish a new wire format. If another version will NOT publish anything substantial, it could be skipped. Equally if another version adds 2, both could be mentioned in this doc.

Counter argument is that proto is spec, but this needs more human explanation as we do with old proto, so perhaps here makes sense?

Reply:
Assuming we agree we put proto explanation here in this spec.